app/Repositories/WorkspaceRepository.php

```diff
@@ -83,9 +111,9 @@ public function pluckWorkspace()
         return $this->model->pluck('name', 'id');
     }
 
-    public function getOneDate($workspace_id, $date, $filter = [])
+    public function getOneDate($workspaceId, $date, $filter = [])
     {
-        $workspace = $this->model->findOrFail($workspace_id);
+        $workspace = $this->model->findOrFail($workspaceId);
         if (!$workspace || !$date) {
             return null;
         }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/WorkspaceRepository.php</br>Whitespace found at end of line